mlflow/java/spark/pom.xml

```diff
@@ -43,12 +43,6 @@
     </dependency>
 
     <!-- Test -->
-    <dependency>
```

Comment:
Removing this as we don't test against Delta here, as Delta Lake doesn't yet work with Spark 3.0 (see https://stackoverflow.com/questions/59116065/spark-3-0-not-able-to-save-a-df-as-delta-table,)